packages/@react-stately/selection/src/useMultipleSelectionState.ts

```diff
@@ -16,9 +16,25 @@ import {MultipleSelectionState} from './types';
 import {Selection} from './Selection';
 import {useControlledState} from '@react-stately/utils';
 
+function equalSets(setA, setB) {
+  if (setA.size !== setB.size) {
+    return false;
+  }
+
+  for (let item of setA) {
+    if (!setB.has(item)) {
+      return false;
+    }
+  }
+
+  return true;
+}
+
 export interface MultipleSelectionStateProps extends MultipleSelection {
   /** How multiple selection should behave in the collection. */
-  selectionBehavior?: SelectionBehavior
+  selectionBehavior?: SelectionBehavior,
+  /** Whether onSelectionChange should always be fired, even if the key is the same. */
+  alwaysFireOnSelection?: boolean
```

Comment:
Name suggestions welcome

Reply:
`allowDuplicateSelectionEvents` is closer to following the naming guidelines, but a little less explicit. wdyt?

Reply:
Sounds good to me, the description should hopefully bring enough clarity as to what the prop does.